maestro-client/src/main/java/maestro/Driver.kt

```diff
@@ -84,4 +84,5 @@ interface Driver {
 
     fun isUnicodeInputSupported(): Boolean
 
+    fun isScreenChanged(): Boolean
```

Comment:
Lets call this `isScreenStatic`, also `screenDiff` is better named `isScreenStatic`.